cdap-ui/app/features/search/templates/search-objects-with-tags.html

```diff
@@ -15,95 +15,101 @@
 -->
 
 <my-breadcrumbs></my-breadcrumbs>
-<div class="row">
-  <div class="col-sm-6">
-    <h2>
-      <span>Search</span>
-    </h2>
-  </div>
-  <div class="col-sm-6 text-right">
-  </div>
-</div>
 
 <div class="row main-search">
-  <div class="col-xs-4">
-    <div class="input-group pull-right">
-      <div class="input-group-addon">
-        <span class="fa fa-search"></span>
-      </div>
-      <input type="text"
-          ng-model="SearchObjectWithTagsController.searchTxt"
-          cask-focus="searchObjectWithTags"
-          class="form-control">
-    </div>
-  </div>
-  <div class="col-xs-12 h2">
-
-      <div ng-repeat="tObject in SearchController.filteredTObjects = (SearchObjectWithTagsController.taggedObjects | filter: SearchObjectWithTagsController.searchTxt ) ">
-        <div ng-if="tObject.type === 'application'">
-          <a class="btn btn-default"
-              ui-sref="apps.detail.overview.status({namespace: tObject.namespaceId, appId: tObject.appId})"
-          >
-            <span class="{{tObject.icon}}"></span>
+  <div class="col-xs-12">
+    <p class="results-count" ng-if="SearchController.filteredTObjects.length">
+      <span>{{SearchController.filteredTObjects.length}}</span>
```

Comment:
Is it possible for us to use ```ngPluralize``` here (https://docs.angularjs.org/api/ng/directive/ngPluralize)?